src/Symfony/Component/HttpKernel/EventListener/LocaleListener.php

```diff
@@ -42,10 +42,14 @@ public function __construct(RequestStack $requestStack, $defaultLocale = 'en', R
         $this->router = $router;
     }
 
+    public function onKernelRequestEarly(GetResponseEvent $event)
```

Comment:
I would change the name to something like `setDefaultLocale()`. There is no `KernelRequestEarly` event thus having this method name could be confusing when reading the code.